spec/features/users/logout_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Logging in', type: :feature do
+  describe 'As a regular User' do
+    it 'can login with valid credentials' do
+      mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+      paper = mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 25)
+      user_1 = create(:user, role: 0)
+
+      visit '/'
+      click_link('Login')
+      fill_in :email, with: user_1.email
+      fill_in :password, with: 'password1'
+      click_on('Submit')
```

Comment:
This login process can be stubbed with   `allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_1)`

Reply:
In order to create real session(s), it looks like we need to go through the motions so sessions are created and destroyed.